requirements.txt

```diff
@@ -1,6 +1,6 @@
 asgiref==3.2.10
 Django==3.1.2
-djangorestframework==3.12.1
+djangorestframework==3.12.2
```

Comment:
Updated in hopes it would fix a bug.  It didn't (the bug was mine), but it didn't cause any new bugs, either.